src/ui/DynamicFacetManager/DynamicFacetManager.ts

```diff
@@ -71,7 +72,18 @@ export class DynamicFacetManager extends Component {
      */
     compareFacets: ComponentOptions.buildCustomOption<IDynamicFacetManagerCompareFacet>(() => {
       return null;
-    })
+    }),
+    /**
+     * Defines the maximum number of expanded facets inside the manager.
+     * The rest of the facets will be collapsed.
+     *
+     * **Note:**
+     * Will prioritize the facets that possess active values, then will prioritize the first facets.
```

Comment:
Suggestion: Prioritizes facets with active values, and then prioritizes first facets.